### DISCUSSION TITLE
Why gitignore files wont show?

### DISCUSSION THREAD

**Discussion by SchoolyB** [nvim-tree/nvim-tree.lua#2577] [Q&A] [answered]
Nvim noob here

I don't understand why some files in my .gitignore doesn't show in the tree. Just because I don't want to push these files to remote doesn't mean I don't want to work on them locally. Tried using `:help nvim......` and nothing comes up

Anyone know how to fix this?

**Comment by gegoune** [comment, marked as answer]
Git ignored files are filtered out by default. Search for `git_ignored` in `:h nvim-tree-setup` to disable filtering. You can also toggle it with `I`, have a look at `:h nvim-tree-opts-filters` .

**Reply by SchoolyB** [reply]
@gegoune Thank you so much!
